run.py

```diff
@@ -106,6 +126,19 @@ def config_validator():
         app.config['TESTING'] = True
         print("Starting server in ~*TEST MODE*~")
         app.run(host='localhost')
+    elif args.backup:
+        try:
+            command = ['ls', '-l', 'issues.db']
+            if execute_command(command)[1] == 0:
+                execute_command(['mkdir', '-p', app.config['BACKUP_DEFAULT_DEST']])
+                current_ts = time.time()
+                time_stamp = datetime.datetime.fromtimestamp(current_ts).strftime('%Y-%m-%d-%H:%M:%S')
+                issue_backup_db = 'issues_backup_' + time_stamp + '.db'
+                execute_command(['mv', 'issues.db', app.config['BACKUP_DEFAULT_DEST'] + issue_backup_db])
+            else:
+                print FILE_NOT_FOUND_ERROR
+        except OSError, e:
+            raise OSError('{0}\n\n{1}'.format(e, FILE_NOT_FOUND_ERROR))
```

Comment:
We can probably just `sys.exit(FILE_NOT_FOUND_ERROR)` if we didn't find the file using os.path.exists.